test/OpenTelemetry.Instrumentation.SqlClient.Tests/SqlClientIntegrationTests.cs

```diff
@@ -0,0 +1,131 @@
+﻿// <copyright file="SqlClientIntegrationTests.cs" company="OpenTelemetry Authors">
```

Comment:
Our dotnet-format CI is failing on this file. It's got a BOM that needs to be removed. File format should be plain UTF-8

Reply:
bad 👮‍♂️

Reply:
Call the BOM squad!

Reply:
I saved the file without BOM.